assets/robots.txt

```diff
@@ -0,0 +1 @@
+User-agent: *
```

Comment:
Always create the robots.txt file?
If it's not going to have anything, better not to have it, no?

Reply:
IMO you could put this in the same place where the `Disallow` gets added. But, optional.

Reply:
Done.